common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -236,14 +175,14 @@ $link-color: rgb(26, 161, 222) !default;
           }
 
           &::after {
-            background: #333;
+            @include transform(rotate(45deg));
+            @include right(18px);
```

Comment:
This right position doesn't look correct in RTL. As it flips to left in RTL, it actually points to the neighbor nav.
<img width="433" alt="screen shot 2016-10-03 at 2 20 55 pm" src="https://cloud.githubusercontent.com/assets/16127998/19048521/ab770178-8974-11e6-8f68-701c7ca79d85.png">